Simplenote/src/debug/AndroidManifest.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<manifest package="com.automattic.simplenote"
```

Comment:
```suggestion
<manifest package="com.automattic.simplenote.debug"
```

The package for the `debug` build type is `com.automattic.simplenote.debug` since the `build.gradle` adds the `.debug` suffix [here](https://github.com/Automattic/simplenote-android/blob/5977d521b6c1b6bfebd1c97e8595c8f20dcf4589/Simplenote/build.gradle#L9-L11).  Is that required for the screenshot automation?

Reply:
I tried with `.debug` originally, but got this error

```
Manifest merger failed :

Overlay manifest:package attribute declared at AndroidManifest.xml:3:11-52
value=(com.automattic.simplenote.debug) has a different
value=(com.automattic.simplenote.debug) declared in main manifest at
AndroidManifest.xml:4:5-40

Suggestion: remove the overlay declaration at AndroidManifest.xml and place it in
the build.gradle:

		flavorName {
			applicationId = "com.automattic.simplenote.debug"
		}
```